install-and-configure/install/helm-install-params.md

```diff
@@ -1,39 +1,42 @@
 # Helm Parameters
 
-There are three different approaches for passing custom Helm config values into the Kubecost project:
-
-1.  **Pass exact parameters via `--set` command-line flags.** For example, you can only pass a product key if that is all you need to configure.
-
-    ```bash
-    $ helm install kubecost cost-analyzer \
-        --repo https://kubecost.github.io/cost-analyzer/ \
-        --namespace kubecost --create-namespace \
-        --set kubecostProductConfigs.productKey.key="123"
-        ...
-    ```
-2.  **Pass exact parameters via custom `values` file.** Similar to option #1, you can create a separate values file that contains only the parameters needed.
-
-    ```bash
-    $ helm install kubecost cost-analyzer \
-        --repo https://kubecost.github.io/cost-analyzer/ \
-        --namespace kubecost --create-namespace \
-        --values values.yaml
-    ```
-
-    _values.yaml:_
-
-    ```yaml
-    kubecostProductConfigs:
-      productKey: 
-        key: "123"
-        enabled: true
-    ```
-3. **Use** [**values.yaml**](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml) **from the Kubecost Helm Chart repository.**
-
-{% hint style="info" %}
+Often while using and configuring Kubecost, our documentation may ask you to pass certain Helm flag values. There are three different approaches for passing custom Helm values into your Kubecost product, which are explained in this doc. In these examples, we are updating the `kubecostProductConfigs.productKey.key` Helm value which enables Kubecost Enterprise, however these methods will work for all other Helm flags.
+
+## Method 1: Pass exact parameters via `--set` command-line flags
+
+For example, you can only pass a product key if that is all you need to configure.
+
+```bash
+$ helm install kubecost cost-analyzer \
+    --repo https://kubecost.github.io/cost-analyzer/ \
+    --namespace kubecost --create-namespace \
+    --set kubecostProductConfigs.productKey.key="123"
+    ...
+```
+## Method 2: Pass exact parameters via custom `values` file
+
+Similar to Method 1, you can create a separate values file that contains only the parameters needed.
+
+Your _values.yaml_ should look like this:
+
+```
+kubecostProductConfigs:
+  productKey: 
+    key: "123"
+    enabled: true
+```
+
+Then run your install command:
+
+```
+$ helm install kubecost cost-analyzer \
+    --repo https://kubecost.github.io/cost-analyzer/ \
+    --namespace kubecost --create-namespace \
+    --values values.yaml
+```
+
+## Method 3: Use [_values.yaml_](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml) from the Kubecost Helm chart repository.
```

Comment:
Do we need to keep this? The referenced values file contains the default values applied when running a helm install. You don't need to refer to that file specifically unless the user has modified it.

Reply:
@jessegoodier @chipzoller What are your thoughts?